plugins/colorize-applied.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: colorize-applied
+spec:
+  version: v0.0.1
+  homepage: https://github.com/cappyzawa/kubectl-colorize-applied
+  shortDescription: "Colorize the applied result"
+  description: |
+    Enjoy coloring the applied results.
```

Comment:
This doesn't help with anything. 

Please explain what the plugin does as if the screenshot doesn't exist.

Reply:
Thank you for your review!
I fixed it. (https://github.com/kubernetes-sigs/krew-index/pull/2522/commits/28f35b88ef0ffbd8e5421b160b8ff88a9df5d149) .